### ISSUE TITLE
Crashes and white screen with error

### ISSUE THREAD

**Issue by patel934** [ryanguill/roam-depot-export-formatter#19]
I'm not sure what I'm doing wrong, but I can't use this promising extension because Roam misbehaves when I click the button to export or the Command Palette. I see a white screen with an error and also drop-down menus to export. But this page doesn't work and I've experienced multiple crashes (desktop app, macOS). I think this extension needs some updating.

**Comment by ryanguill** [comment]
Sorry to hear that. Things seem to continue to work for me.
Can you provide the error message you get or a screenshot (or even better yet a screen recording so I can see exactly what youre doing?)

One thing to keep in mind is that you want to right click on the node above the item(s) you are trying to format, not a single node with content.

**Comment by patel934** [comment]
<img width="859" alt="Screenshot 2024-11-25 at 20 05 44" src="https://github.com/user-attachments/assets/c0bbd8f4-7ae4-4cad-a544-7799e60b5d5a">

The screen looked this this. I was actually clicking the Folder Icon in the top bar in Roam and getting this error. Now, I right-clicked the node and clicked Extensions. Then I was able to get it working! I already did this manually a while ago, but guess it'll save me time in the future. Very good to know! The folder icon keeps giving me problems...
